languages/de.json

```diff
@@ -1290,6 +1362,7 @@
 "DND5E.SavePromptTitle": "{ability} Rettungswurf",
 "DND5E.SavingThrow": "Rettungswurf",
 "DND5E.SavingThrowDC": "SG {dc} {ability}srettungswurf",
+"DND5E.SavingThrowRoll": "Werfe {ability}srettungswurf",
```

Comment:
Oh, if we just had more powerful translation features. "Intelligenz**s**rettungswurf" unfortunately is wrong. We could do "{ability}-Rettungswurf" for "Intelligenz-Rettungswurf", maybe. Anyway, I note the existing translation in the line above already does this also, so we can fix this later. No worries for now.